pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,111 @@
+# Test ownership
+
+Customers on the [Pro and Enterprise plans](https://buildkite.com/pricing) can assign test ownership to [Teams](/docs/team-management/permissions).
+
+Test ownership is assigned to the tests in a specific suite per instructions in your TESTOWNER file. The team that is the default owner of a test [will be auto-assigned to fix any flakes](/docs/test-analytics/flaky-test-assignment) that occur.
```

Comment:
What do you think about dropping the _per-suite_ instructions here, and we can talk about that requirement later in the docs... mainly because there's nothing stopping someone using the same file uploaded to multiple suites. 

Something like this perhaps:

```suggestion
Test ownership is managed via team assignments in a TESTOWNER file. The team that is the default owner of a test [will be automatically assigned flaky tests](/docs/test-analytics/flaky-test-assignment) to triage.
```

Reply:
Thanks I'll use that!